spec/rails_helper.rb

```diff
@@ -0,0 +1,19 @@
+ENV['RAILS_ENV'] ||= 'test'
```

Comment:
Aren't you supposed to add coveralls settings here? I see that you included the gem, so I assume that you want to set it up as well....

Reply:
Yes we are and I was sure we did that, but apparently not. Apologies for that. It's added now